angular-client/src/components/carousel/carousel.component.html

```diff
@@ -26,9 +28,15 @@
           </mat-grid-tile>
         </mat-grid-list>
       </div>
-      <div class="select-button-container">
-        <button mat-button class="run-select-button" (click)="selectRun(run)">Select this Run</button>
-      </div>
     </ng-template>
   </p-carousel>
-</div>
+  
+  <div class="select-number-run">
+    <button *ngFor="let run of runs; let idx = index" 
+            mat-button 
+            class="number_buttons" 
+            [ngClass]="{'highlight': currentIndex === idx}" 
+            (click)="selectRun(run, idx)">
```

Comment:
clicking this should show the run in the carousel, you need a seperate button for selecting the run